tests/functional/p/possible_forgotten_f_prefix.py

```diff
@@ -0,0 +1,38 @@
+# pylint: disable=missing-docstring, invalid-name, line-too-long, multiple-statements, pointless-string-statement, pointless-statement
+var = "string"
+var_two = "extra string"
+
+x = f"This is a {var} which should be a f-string"
+x = "This is a {var} used twice, see {var}"
+x = "This is a {var} which should be a f-string"  # [possible-forgotten-f-prefix]
+x = "This is a {var} and {var_two} which should be a f-string"  # [possible-forgotten-f-prefix, possible-forgotten-f-prefix]
+x1, x2, x3 = (1, 2, "This is a {var} which should be a f-string")  # [possible-forgotten-f-prefix]
+
+y = "This is a {var} used for formatting later"  # [possible-forgotten-f-prefix]
```

Comment:
This should not raise the message, as it is used later in a `format()` call. However, you changed this, so perhaps I am missing a reason why this should raise?

Reply:
At first I thought it was acceptable (contrary to another_var where the var do not exist in the local scope). But I changed my mind, and I think you're right.